### DISCUSSION TITLE
2300 MHz frequency Inquiry

### DISCUSSION THREAD

**Discussion by justinmarkr** [srsran/srsRAN_4G#1355] [General]
Hi srs community,

Does the srsRAN 4G support 2300 MHz frequency for LTE networks? Thank you.

**Comment by Monsieur-ugur** [comment]
Good morning.

srsRAN Project and srsRAN 4G should both support those frequency bands, in your config file there is a field named earfcn (for 4G) or arfcn (for 5G) this code give off both your downlink and uplink frequency. 

From memory the only supported frequency band in the 2300MHz is the "band 30" (in both 4G and 5G). the range of frequency for the uplink is : 2305 MHz – 2315 MHz and  2350 MHz – 2360 MHz for the downlink.  

Here is a link with all of the frequency bands defined by 3GPP and a calculator for earfcn/arfcn : 
https://5g-tools.com/4g-lte-earfcn-calculator/

Wishing you a nice day.
